cardano-cli/cardano-cli.cabal

```diff
@@ -123,6 +123,8 @@ library
                         Cardano.CLI.Types.Common
                         Cardano.CLI.Types.Governance
                         Cardano.CLI.Types.Key
+                        Cardano.CLI.Types.Key.AnyVerificationKeyText
```

Comment:
Separate modules for this is overkill. `Cardano.CLI.Types.Key.VerificationKey` is enough.